client/app/bundles/App/GettingStarted/Theming.md

```diff
@@ -1,3 +1,103 @@
 ## Theming
 
-We are currently updating all of our components to use CSS modules / CSS custom properties. More information on theming will be available after that work has been completed.
+Terra UI provides theming capabilities in various components via [CSS custom properties](https://developer.mozilla.org/en-US/docs/Web/CSS/--*).
+
+### Terra default theme
+Terra UI provides a default theme for components via [CSS custom properties](https://developer.mozilla.org/en-US/docs/Web/CSS/--*).
+For teams that do not have additional theming needs outside of the default terra theme, we recommend using the [PostCSS loader for webpack ](https://github.com/postcss/postcss-loader/) along with the [CustomProperties plugin](https://github.com/postcss/postcss-custom-properties) to compile CSS custom properties down to static values. This ensures the theme styles render in modern browsers as well as those which do not support CSS custom properties.
```

Comment:
nits:
- remove additional (For teams that do not have **additional** theming)
- add comma (...styles render in modern browsers**,** as well as those..)

- Should we explicitly list the modern browsers that do and do not support CSS custom properties or provided reference to what these browsers are?

Reply:
Updated: https://github.com/cerner/terra-ui.com/pull/53/commits/db2238d6f8baef77144add2e1df6248b460be568